### PR TITLE
Add `split_4x6_sufl` layout

### PR DESCRIPTION
**Pull request by dunk2k**
## Description
- Create community layout for Sofle layout keyboards 
- Move `sofle/keymaps/manna-harbour_miryoku` into `layouts/community/split_4x6_sufl`
- apply `LAYOUT_split_4x6_sufl` and `community_layout` to:
    - keycapsss/kimiko
    - sofle
    - splitkb/aurora/sofle_v2
    - tkw/grandiceps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — COMMENTED** [review]
not really a fan of the naming here.  Generally, want to keep it generic/hardware agnostic naming.

**Comment by dunk2k** [comment]
@manna-harbour 
as this PR adds miryoku for new keyboards but changes the method in how miryoku is applied (i.e. keymap to QMK/community layout) for a small number of existing keyboards, should these changes be reflected in `users/manna-harbour-miryou/readme.org` of this PR?

**Comment by drashna** [comment]
> I'm open to suggestions 🙂

Honestly, I'm absolutely horrible at naming.  But the idea is to not have the keyboard name as the layout name

File: users/manna-harbour_miryoku/readme.org [hunk 1]
```diff
@@ -459,6 +459,18 @@ make bastardkb/tbkmini:manna-harbour_miryoku:flash # bastardkb/tbkmini
 make crkbd:manna-harbour_miryoku:flash # crkbd
 #+END_SRC
 
+**** split_4x6_sofl
+
+For each half: the top row, outer column, inner encoder, and 2 outward most keys on bottom row are unused.
```
Comment (manna-harbour) [inline users/manna-harbour_miryoku/readme.org:464]:
```suggestion
Only the main 5x3 alphas and the inner 3 thumb keys are used.
```
For consistency.

**Comment by manna-harbour** [comment]
> @manna-harbour as this PR adds miryoku for new keyboards but changes the method in how miryoku is applied ) for a small number of existing keyboards (i.e. keymap to QMK/community layout), should these changes be reflected in `users/manna-harbour-miryou/readme.org` of this PR?

Thanks for your update! I've suggested a small change to the readme but it otherwise looks fine to me.

Just a general note for anyone's future PRs:

1. In general, I think changes shouldn't be made to other user's keymaps without consulting first. In this case you've pinged me in the PR so that's ok.

2. Specifically, for changes to files that are part of Miryoku QMK, the changes should be made in a PR to the Miryoku QMK repo. Those changes will in turn be PR'd upstream to QMK in the next periodic update PR. If changes are made directly in QMK they may introduce merge conflicts with the `miryoku` branch, breaking the Miryoku QMK build workflow automatic branch merge feature. In this case, if it was done as separate PRs first to QMK without the Miryoku files, and then to Miryoku QMK with only the Miryoku files, Miryoku QMK would need to be rebased first anyway, and also I'm not planning on making changes to the areas you've touched without rebasing first, so doing it as one PR directly in QMK is ok.

**Comment by dunk2k** [comment]
Closing PR:
questionable is the actual usage of this community layout, considering current/established community layouts aren't utilised by userbase and it's arguable how well known by userbase Community Layout feature is.

**Pull request closed without merging**